File: Chatbot WCF IIS HOST/wcf 2/Chatbot/Models/Chart.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Chatbot.Models
{
    public class Chart
    {
        public string type { get; set; }
        public List<string> labels = new List<string>();
        public List<ChartData> dataSets = new List<ChartData>();
    }
}

File: Chatbot Angular/Chatbot Desktop Version/Chat/src/app/classes/Chart.ts
export class ChartData{
    data:number[] = [];
    label : string;
    constructor(){}
}

export class Chart{
  type : string;
  labels : string[] = [];
  options : any = { legend: { display: true, responsive : true,maintainAspectRatio:false}};
  colors : Array<any> = [];
  dataSets : ChartData[] = [];
}